packages/esm-home-app/src/openmrs-esm-home-schema.ts

```diff
@@ -41,4 +41,39 @@ export const esmHomeSchema = {
       _description: 'Whether to show widgets on the home page (including extensions)',
     },
   },
+  externalRefLinks: {
+    enabled: {
+      _type: Type.Boolean,
+      _default: true,
+      _description: 'Whether to show external links in the app menu',
+    },
+    links: {
+      _type: Type.Array,
+      _elements: {
+        title: {
+          _type: Type.String,
+          _description: 'Title of the link',
+        },
+        redirect: {
+          _type: Type.String,
+          _description: 'Link to redirect to (must be an external link)',
+        },
+      },
+      _default: [
+        {
+          title: 'Pharmacy',
```

Comment:
The problem with such an approach is that you cant easily disable just a single entry. If you want to disable, e.g., "Pharmacy" you'll need to override everything (i.e., define all the links, in this case "Laboratory" and "Analytics"). If that's desired its fine, otherwise I'd recommend a different approach.

Reply:
HI @FlorianRappl , I get your point. What is the different approach you are talking about?